### ISSUE TITLE
6.2.60 Build error: "cannot find symbol androidx.annotation.NonNull"

### ISSUE THREAD

**Issue by iZerus** [AppsFlyerSDK/appsflyer-cordova-plugin#165]
# Report

## Plugin Version

 `6.2.60`

## On what Platform are you having the issue?

android

## What did you do?

Cmd: `cordova build android`

## What did you expect to happen?

App to be builded

## What happened instead?

Error:
```log
D:\Cordova\app\cordova\platforms\android\app\src\main\java\com\appsflyer\cordova\plugin\AppsFlyerPlugin.java:37: error: cannot find symbol
import androidx.annotation.NonNull;
                          ^
  symbol:   class NonNull
  location: package androidx.annotation
> Task :app:compileDebugJavaWithJavac FAILED
24 actionable tasks: 24 executed
D:\Cordova\app\cordova\platforms\android\app\src\main\java\com\appsflyer\cordova\plugin\AppsFlyerPlugin.java:282: error: cannot find symbol
            public void onDeepLinking(@NonNull DeepLinkResult deepLinkResult) {
                                       ^
  symbol: class NonNull
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

FAILURE: Build failed with an exception.
```

**Comment by amit-kremer93** [comment]
@iZerus did you try to remove android platform and add it again?

**Comment by amit-kremer93** [comment]
Problem fixed in 6.3.0
